packages/runtime/container-runtime/src/test/summary/summarizerHeuristics.spec.ts

```diff
@@ -231,20 +231,18 @@ describe("Runtime", () => {
 			});
 
 			it("Should summarize on close if enough outstanding ops", () => {
-				const lastSummary = 1000;
 				const minOpsForLastSummaryAttempt = 10;
-				initialize({ refSequenceNumber: lastSummary, minOpsForLastSummaryAttempt });
+				initialize({ minOpsForLastSummaryAttempt, runtimeOpWeight: 1.0 });
```

Comment:
You should add tests where both types of ops exist with different weights. Then validate that summarize on close runs accordingly.
For instance, 10 non-user ops should not trigger summary which is what would happen without this change.